main.go

```diff
@@ -35,16 +41,61 @@ func main() {
 	fmt.Println()
 	log.Infof("Running test")
 
-	testCmd := command.New("flutter", append([]string{"test"}, additionalParams...)...).
+	pr, pw := io.Pipe()
+
+	if _, err := exec.LookPath("tojunit"); err != nil {
+		log.Infof("Command `tojunit` not found, installing...")
+		junitInstallCmd := command.New("flutter", append([]string{"pub", "global", "activate", "junitreport"})...).
+			SetStdout(os.Stdout).
+			SetStderr(os.Stderr).
+			SetDir(cfg.ProjectLocation)
+		if err := junitInstallCmd.Run(); err != nil {
+			failf("Command `tojunit` failed to install, error: %s", err)
+		}
+	}
+
+	testCmd := exec.Command("flutter", append([]string{"test", "--machine"}, additionalParams...)...)
```

Comment:
Please pirint the ran command ( `flutter test --machine ... | tojunit --ouptut ...`

Reply:
The command package does not support Start & Wait so piping the commands together would cause a deadlock